### DISCUSSION TITLE
Configuring Node's color

### DISCUSSION THREAD

**Discussion by JpEncausse** [node-red-contrib-themes/theme-collection#530] [General]
Hi,
I correctly installed in v4.0.3 `node-red-contrib-themes` with `github-dark-dimmed` it works great.

I was wondering how/if I can change Node's color ? In Dark themes they looks bright and the gray border doesn't look sharp with the background.

Is it a feature of the module ? Or something I could change on the settings ? Or is there a complementary module ?

Thanks !

**Comment by JpEncausse** [comment]
I found a work around using a custom css file with the following code.

```
.red-ui-flow-node {
    stroke-width: 1;
    stroke: #222;
    filter: saturate(1.2) brightness(0.9) drop-shadow(0px 0px 1px #FFF);
}
```

**Comment by bonanitech** [comment]
We don't change node colors in themes. Their colors are defined by their authors.

Also, there's this recommendation from Node-RED's creator.

![c3487e7ada917a49df4d7a8485e8dffd64fe5091](https://github.com/user-attachments/assets/fe4e6383-967a-40a7-b5cf-87854063666d)

You can write your customizations to a file and add it to your settings.

See https://nodered.org/docs/user-guide/runtime/configuration#editor-themes for reference.
